demisto_sdk/commands/lint/linter.py

```diff
@@ -521,11 +523,12 @@ def _run_vulture(self, py_num: float, lint_files: List[Path]) -> Tuple[int, str]
 
         return SUCCESS, ""
 
-    def _run_lint_on_docker_image(self, no_pylint: bool, no_test: bool, no_pwsh_analyze: bool, no_pwsh_test: bool,
+    def _run_lint_on_docker_image(self, no_mypy: bool, no_pylint: bool, no_test: bool, no_pwsh_analyze: bool, no_pwsh_test: bool,
```

Comment:
it's a bad practice to add it to the start of this function/ add it to the end.
it may break something else

Reply:
since there are no optional values involved, any call to this function will break anyway.
better to keep all the `no_*` linters together as they share a purpose.